graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/config/DXGraphQLConfig.java

```diff
@@ -131,6 +141,10 @@ public void deleted(String pid) {
         }
     }
 
+    public boolean isIntrospectionEnabled() {
+        return SettingsBean.getInstance().isDevelopmentMode() || introspectionEnabled;
```

Comment:
Just wondering about this statement, what would happen if `graphql.introspection.enabled = false`when running in development ? 
In such a case, introspection should instead be disabled.

Reply:
ok I was under the impression we wanted it enabled for development all the time and that the flag only affects production. 

In this case, we can add a separate property specifically for dev and enable that by default.

Reply:
After discussion, found a way to reuse the same property and apply different default values for dev and prod if not specified in the default configuration.